apps/experiments/views/experiment.py

```diff
@@ -586,6 +594,15 @@ def _start_experiment_session(
             status=session_status,
             participant=participant,
         )
+
+        # Record the participant's timezone
+        if timezone:
+            data_records = ParticipantData.objects.filter(participant=participant)
+            for data_record in data_records:
+                data_record.data["timezone"] = timezone
+                data_record.save()
+            ParticipantData.objects.bulk_update(data_records, fields=["data"])
```

Comment:
We cannot do that, since the `data` field is not json, but rather an encrypted version of the json. Since decoding happens at the application layer, we sadly cannot do these types of DB operations